### PR TITLE
NEWS: update for 2.0.2 release

### LINKED ISSUE #2239: update NEWS for v2.0.2 release
Update NEWS for v2.0.2 release

### PR DESCRIPTION
**Pull request by hppritcha**
Fixes #2239

@jsquyres 

Signed-off-by: Howard Pritchard <howardp@lanl.gov>

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -53,6 +53,37 @@ included in the vX.Y.Z section and be denoted as:
...
+  enabled.  Thanks to amckinstry for reporting.
+- Fix a problem using MPI_IALLTOALL with MPI_IN_PLACE argument.  Thanks to
+  Chris Ward for reporting.
+- Fix a problem using MPI_RACCUMULATE with Portals4 transport.  Thanks to
+  PDeveze for reporting.
+- Fix an issue with static linking and duplicate symbols.  Thanks to
+  Limin Gu for reporting.
+- Fix a problem when using MPI dynamic memory windows.  Thanks to
+  Christoph Niethammer for reporting.
+- Fix a problem with Open MPI's pkgconfig files.  Thanks to amckinstry
```
Comment (jsquyres) [inline NEWS]: @amckinstry is Alastair McKinstry; let's use his name.

**Comment by hppritcha** [comment]
@jsquyres put in Alastair's full name, added some new bullets for more recent commits reported by users, added some verbosity to a some of the bullets to help with google search.

**Comment by hppritcha** [comment]
I changed the Valgrind spelling somewhat.  Capitalized if used as a proper noun, otherwise, lower case if used as an adjective or adverb.

**Comment by hppritcha** [comment]
updated with info relevant to PR #2529

**Comment by hppritcha** [comment]
@jsquyres updated to include PR #2556

**Comment by hppritcha** [comment]
Have to careful with commits from the web interface to manually add in a sign off line.

**Comment by jsquyres** [comment]
@hppritcha Did you want to squash these down?

**Pull request merged**